src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -282,38 +381,17 @@ def submitJob(self, executableFile, proxy, numberOfJobs=1):
         batchIDList = []
         stampDict = {}
         for _ in range(numberOfJobs):
-            # Get the job into the ARC way
-            xrslString, diracStamp = self._writeXRSL(executableFile)
-            xrslString += delegation
-            self.log.debug("XRSL string submitted", "is %s" % xrslString)
-            self.log.debug("DIRAC stamp for job", "is %s" % diracStamp)
-
-            # Submit the POST request
-            response = self.session.post(
-                query,
-                data=xrslString,
-                headers=self.headers,
-                params=params,
-                timeout=self.arcRESTTimeout,
-            )
-            if not response.ok:
-                self.log.warn(
-                    "Failed to submit job",
-                    "to CE %s with error - %s - and messages : %s"
-                    % (self.ceHost, response.status_code, response.reason),
-                )
+            result = self._getJobID(executableFile, inputs, outputs, executables, delegation)
+            if not result["OK"]:
                 break
```

Comment:
```suggestion
            if not result["OK"]:
                self.log.error(result["Message"])
                break
```
or something better.